mix.exs

```diff
@@ -73,13 +73,17 @@ defmodule BitcrowdEcto.MixProject do
     [
       {:ecto, "~> 3.6"},
       {:ecto_sql, "~> 3.6"},
+      {:ex_money, "~> 5.12", only: [:dev, :test], optional: true},
       {:credo, "~> 1.6", only: [:dev, :test], runtime: false},
       {:dialyxir, "~> 1.1", only: [:dev, :test], runtime: false},
       {:ex_doc, "> 0.0.0", only: [:dev], runtime: false},
       {:ex_machina, "~> 2.7", only: [:dev, :test]},
       {:junit_formatter, "~> 3.3", only: [:test]},
       {:postgrex, "> 0.0.0", only: [:dev, :test]},
-      {:tzdata, "> 0.0.0", only: [:dev, :test]}
+      {:tzdata, "> 0.0.0", only: [:dev, :test]},
+      {:ex_cldr, "~> 2.33", only: [:dev, :test]},
+      # https://github.com/kipcole9/money/issues/142
+      {:ex_money_sql, "~> 1.7", only: [:dev, :test], override: true}
```

Comment:
is setting override in a lib also overriding the version of the parent project including it?

Reply:
no idea :) I hope not

Reply:
If it works I am fine with that 🤷 
Having `ex_money_sql` as `optional: true` would also have made sense to me....

Reply:
It works, yeah, but IMO this solution expresses the actual dependencies better. I'd leave it like this, if you're ok with it.